### PR TITLE
DOC: Clean up `itk::ConformalFlatteningMeshFilter` Doxygen documentation

### PR DESCRIPTION
**Pull request by jhlegarreta**
Clean up method Doxygen documentation for
`itk::ConformalFlatteningMeshFilter`:
- Consolidate documentation so that methods are documented in the header files.
- Remove empty method documentation blocks from implementation file.

Take advantage of the commit to make the modified method documentation
blocks have a consistent line length.

Follow-up to d58bf56fe2320fe3c7a451c96e3d8cd5e7c68358.

## PR Checklist
- [X] No [API changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#breaking-changes) were made (or the changes have been approved)
- [X] No [major design changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#design-changes) were made (or the changes have been approved)
- [X] Updated API documentation (or API not changed)

### CODE REVIEW

File: Modules/Nonunit/Review/include/itkConformalFlatteningMeshFilter.h [hunk 1]
```diff
@@ -89,7 +89,7 @@ class ITK_TEMPLATE_EXPORT ConformalFlatteningMeshFilter : public MeshToMeshFilte
   using PointIdIterator = typename CellType::PointIdIterator;
   using CellAutoPointer = typename CellType::CellAutoPointer;
 
-  /** Select the cell that will be used as reference for the flattening.
+  /** Select the cell that will be used to define the boundary/used as reference for the flattening.
```
Comment (dzenanz) [inline Modules/Nonunit/Review/include/itkConformalFlatteningMeshFilter.h:92]: Maybe reflow this whole paragraph?

**Review by dzenanz — APPROVED** [review]

File: Modules/Nonunit/Review/include/itkConformalFlatteningMeshFilter.h (hunk 1 above)
Comment (jhlegarreta) [reply Modules/Nonunit/Review/include/itkConformalFlatteningMeshFilter.h:92]: Not sure I understand the term "reflow".

Comment (dzenanz) [reply Modules/Nonunit/Review/include/itkConformalFlatteningMeshFilter.h:92]: Align it to about the same column width. This is a nitpick, not a requirement for merging.

Comment (jhlegarreta) [reply Modules/Nonunit/Review/include/itkConformalFlatteningMeshFilter.h:92]: :+1: Done in 9b02586ca9687561bd7992c4d890bed769a04b4b.

**Review by hjmjohnson — APPROVED** [review]

**Pull request merged**